aws/aws.go

```diff
@@ -151,6 +177,11 @@ var CNNorth = Region{
 	"https://sdb.cn-north-1.amazonaws.com.cn",
 	"https://sns.cn-north-1.amazonaws.com.cn",
 	"https://sqs.cn-north-1.amazonaws.com.cn",
+	// TODO (anastasiamac 2015-03-05) should this not be
+	// https://iam.cn-north-1.amazonaws.com.cn
+	// ??
+	// I am looking at http://docs.aws.amazon.com/general/latest/gr/isolated_regions.html
```

Comment:
They both resolve to the same IP, but we should probably use the FQDN specified in the docs.